app/lib/cognito.server.ts

```diff
@@ -116,7 +183,14 @@ export async function getGroups() {
   const groups: GroupType[] = []
   for await (const page of pages) {
     const nextGroups = page.Groups
-    if (nextGroups) groups.push(...nextGroups)
+    if (nextGroups)
+      groups.push(
+        ...nextGroups.filter(
+          (group) =>
+            group.GroupName !== undefined &&
+            !group.GroupName.startsWith('us-east-1')
```

Comment:
Actually I think that we should only _include_ groups that start with `gcn.nasa.gov/`. Do we have a constant somewhere that defines that group prefix?

Reply:
What about the igwn alerts?

Reply:
I dont think we do, but i can add one if not

Reply:
I think that group name also starts with gcn.nasa.gov/, doesn’t it?

Reply:
Yep, I was mistaken, should be all good now